apps/test/unit/lib/kits/maker/ThermometerTest.js

```diff
@@ -62,3 +66,74 @@ describe('Thermometer', function() {
     }
   });
 });
+
+describe('MicroBitThermometer', function() {
+  let boardClient;
+  let thermometer;
+
+  beforeEach(() => {
+    boardClient = new MicrobitStubBoard();
+    thermometer = new MicroBitThermometer({mb: boardClient});
+  });
+
+  it(`attributes are readonly`, () => {
+    let attributes = ['raw', 'celsius', 'fahrenheit', 'C', 'F'];
+    let desc;
+
+    attributes.forEach(attr => {
+      desc = Object.getOwnPropertyDescriptor(thermometer, attr);
+      expect(desc.set).to.be.undefined;
+      expect(desc.get).to.be.defined;
+    });
+  });
+
+  it(`fahrenheit is calculated from celsius`, () => {
+    // Seed the temp channel with celsius data
+    boardClient.analogChannel[sensor_channels.tempSensor] = 3;
+
+    expect(thermometer.celsius).to.equal(thermometer.C);
```

Comment:
[not blocking] Why do we have both `thermometer.celsius` and `thermometer.C`? (as well as `.fahrenheit` and `.F`)

Reply:
I was following along with this Johnny Five Thermometer API. However, I noticed that we only have C/F available as blocks. When we make blocks for the MicroBit it would make sense to only make a C/F block to match the CP.